tests/pytest/enrollment_switchio/test_views.py

```diff
@@ -0,0 +1,31 @@
+import json
+import pytest
+
+from benefits.enrollment_switchio.views import GatewayUrlView, IndexView
+
+
+class TestIndexView:
```

Comment:
Having these tests is especially helpful since the views are not actually being used by the application yet. :100:

Reply:
Thanks, yep I used them to confirm the behavior of the new views. I did make a small change to `TestIndexView`. The view itself is now passed as a fixture to the test, anticipating that we'll have more tests that will use it and it's nicer to set it up this way.